src/displayapp/screens/StopWatch.cpp

```diff
@@ -4,49 +4,51 @@
 #include "displayapp/InfiniTimeTheme.h"
 
 using namespace Pinetime::Applications::Screens;
+using namespace Pinetime::Controllers;
 
 namespace {
-  TimeSeparated_t convertTicksToTimeSegments(const TickType_t timeElapsed) {
+  TimeSeparated ConvertTicksToTimeSegments(const TickType_t timeElapsed) {
     // Centiseconds
     const int timeElapsedCentis = timeElapsed * 100 / configTICK_RATE_HZ;
 
     const int hundredths = (timeElapsedCentis % 100);
     const int secs = (timeElapsedCentis / 100) % 60;
     const int mins = ((timeElapsedCentis / 100) / 60) % 60;
     const int hours = ((timeElapsedCentis / 100) / 60) / 60;
-    return TimeSeparated_t {hours, mins, secs, hundredths};
+    return TimeSeparated {hours, mins, secs, hundredths};
```

Comment:
One thing I've noticed: if the stop watch is left running for a long time these calculations overflow. Could this be refactored to avoid anything exceeding a uint32? The actual tick count overflows every `2^32 / 2^10 = 2^22s ~= 48days`, but dividing that by 100 gives just 11.65h (as currently the tick count is multiplied by 100, so overflow happens 100x earlier). I don't think we need to worry about a 48 day overflow, but more than 11h would be nice.